addons/account_asset/account_asset.py

```diff
@@ -1,351 +1,325 @@
-        'company_id': fields.many2one('res.company', 'Company', required=True),
-        'method': fields.selection([('linear','Linear: Computed on basis of Gross Value / Number of Depreciations'),('degressive','Degressive: Computed on basis of Residual Value * Degressive Factor')], 'Computation Method', required=True, help="Choose the method to use to compute the amount of depreciation lines.\n"\
-            "  * Linear: Calculated on basis of: Gross Value / Number of Depreciations\n" \
-            "  * Degressive: Calculated on basis of: Residual Value * Degressive Factor"),
-        'method_number': fields.integer('Number of Depreciations', help="The number of depreciations needed to depreciate your asset"),
-        'method_period': fields.integer('Period Length', help="State here the time between 2 depreciations, in months", required=True),
-        'method_progress_factor': fields.float('Degressive Factor'),
-        'method_time': fields.selection([('number','Number of Depreciations'),('end','Ending Date')], 'Time Method', required=True,
-                                  help="Choose the method to use to compute the dates and number of depreciation lines.\n"\
-                                       "  * Number of Depreciations: Fix the number of depreciation lines and the time between 2 depreciations.\n" \
-                                       "  * Ending Date: Choose the time between 2 depreciations and the date the depreciations won't go beyond."),
-        'method_end': fields.date('Ending date'),
-        'prorata':fields.boolean('Prorata Temporis', help='Indicates that the first depreciation entry for this asset have to be done from the purchase date instead of the first January'),
-        'open_asset': fields.boolean('Skip Draft State', help="Check this if you want to automatically confirm the assets of this category when created by invoices."),
-    }
-
-    _defaults = {
-        'company_id': lambda self, cr, uid, context: self.pool.get('res.company')._company_default_get(cr, uid, 'account.asset.category', context=context),
-        'method': 'linear',
-        'method_number': 5,
-        'method_time': 'number',
-        'method_period': 12,
-        'method_progress_factor': 0.3,
-    }
-
-    def onchange_account_asset(self, cr, uid, ids, account_asset_id, context=None):
-        res = {'value':{}}
-        if account_asset_id:
-           res['value'] = {'account_depreciation_id': account_asset_id}
-        return res
+    active = fields.Boolean(string='Active', default=True)
+    name = fields.Char(required=True, index=True)
+    note = fields.Text()
+    account_analytic_id = fields.Many2one('account.analytic.account', string='Analytic Account')
+    account_asset_id = fields.Many2one('account.account', string='Asset Account', required=True, domain=[('type', '=', 'other')])
+    account_income_recognition_id = fields.Many2one('account.account', string='Recognition Income Account', domain=[('type', '=', 'other')])
```

Comment:
if this field is the new name for account_expense_depreciation_id, then add an oldname parameter (else, ignore this comment)